api/src/graphql/portals.sdl.js

```diff
@@ -0,0 +1,10 @@
+export const schema = gql`
+  type BillingPortalSession {
+    id: String!
+    url: String!
+  }
+
+  type Mutation {
+    createBillingPortalSession(userId: ID!): BillingPortalSession! @requireAuth
+  }
+`
```

Comment:
Renamed this file to `portals`, with an s; just a Redwood convention (for services)